plugins/meson.build

```diff
@@ -43,6 +43,7 @@ plugins = {
   'dell': false,
   'dell-dock': false,
   'dell-kestrel': false,
+  'devlink': false,
```

Comment:
@hughsie thoughts on the plugin name?  As this is 1000% Linux, should this be `linux-devlink`?